File: cdk-from-cfn-testing/expected/bucket/csharp/BucketConstruct.cs
using Amazon.CDK;
using Amazon.CDK.AWS.S3;
using Constructs;
using System.Collections.Generic;

namespace BucketConstruct
{
    public class BucketConstructProps
    {
    }

    public class BucketConstruct : Construct
    {
        public BucketConstruct(Construct scope, string id, BucketConstructProps props = null) : base(scope, id)
        {

            // Resources
            var bucket = new CfnBucket(this, "Bucket", new CfnBucketProps
            {
            });
        }
    }
}

File: cdk-from-cfn-testing/expected/bucket/typescript/BucketConstruct.ts
import * as cdk from 'aws-cdk-lib';
import * as s3 from 'aws-cdk-lib/aws-s3';
import { Construct } from 'constructs';

export interface BucketConstructProps {
}

export class BucketConstruct extends Construct {
  public constructor(scope: Construct, id: string, props: BucketConstructProps = {}) {
    super(scope, id);

    // Resources
    const bucket = new s3.CfnBucket(this, 'Bucket', {
    });
  }
}
